app/src/main/java/org/sopt/and/component/ContentsView.kt

```diff
@@ -0,0 +1,64 @@
+package org.sopt.and.component
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.Info
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.sopt.and.R
+import org.sopt.and.ui.theme.ThirdGrey
+
+@Composable
+fun ContentsView(
+    title: String,
+    contentText: String,
+    titleColor: Color = Color.White,
+    contentTextColor: Color = ThirdGrey,
+    modifier: Modifier
```

Comment:
```suggestion
    modifier: Modifier = Modifier
```
수정자는 일반적으로 기본값을 설정해서 사용합니다!